cmd/entrypoint/main.go

```diff
@@ -96,7 +96,7 @@ func main() {
 	// from secret volume mounts to /tekton/creds. This is done to support the expansion
 	// of a variable, $(credentials.path), that resolves to a single place with all the
 	// stored credentials.
-	builders := []credentials.Builder{dockercreds.NewBuilder(), gitcreds.NewBuilder()}
+	builders := []credwriter.Writer{dockercreds.NewBuilder(), gitcreds.NewBuilder()}
 	for _, c := range builders {
 		if err := c.Write(pipeline.CredsDir); err != nil {
```

Comment:
We should also remove pipeline.CredsDir, may be moved under entrypoint package constants https://github.com/tektoncd/pipeline/blob/main/pkg/entrypoint/entrypointer.go#L55

Reply:
Thank you jawed, just updating the PR with this change now